swarm/api/api_test.go

```diff
@@ -109,11 +109,15 @@ func TestApiPut(t *testing.T) {
 	testAPI(t, func(api *API, toEncrypt bool) {
 		content := "hello"
 		exp := expResponse(content, "text/plain", 0)
-		addr, wait, err := api.Put(context.TODO(), content, exp.MimeType, toEncrypt)
+		ctx := context.TODO()
+		addr, wait, err := api.Put(ctx, content, exp.MimeType, toEncrypt)
```

Comment:
Do we need context in Put as we are using it only in wait(). Also, fileStore.Store uses context only in its wait() function.

Reply:
We are using it to call `fileStore.Store()`, then to call `PyramidSplit()`, then to call `PyramidChunker.Split()`. It is true that `PyramidChunker.Split()` doesn't use the context object yet, that is because this PR is just a preparation to create the API-s for future context usage (which is coming in chunk refactor)

Reply:
Yes, I see, this is just an example for my general question in this PR.